x-pack/plugin/security/qa/multi-cluster/src/javaRestTest/java/org/elasticsearch/xpack/remotecluster/RemoteClusterSecurityCrossClusterApiKeySigningIT.java

```diff
@@ -200,9 +201,16 @@ public void testCrossClusterSearchWithCrossClusterApiKeySigning() throws Excepti
     }
 
     private void assertCrossClusterAuthFail(String expectedMessage) {
+        final long startTimeMillis = System.currentTimeMillis();
         var responseException = assertThrows(ResponseException.class, () -> simpleCrossClusterSearch(randomBoolean()));
         assertThat(responseException.getResponse().getStatusLine().getStatusCode(), equalTo(401));
         assertThat(responseException.getMessage(), containsString(expectedMessage));
+
+        try {
+            assertAuditLogContainsNewEvent(startTimeMillis, "authentication_failed");
+        } catch (Exception e) {
+            throw new RuntimeException("Audit log assertion failed after client-side check passed.", e);
+        }
     }
 
     private void assertCrossClusterSearchSuccessfulWithoutResult() throws IOException {
```

Comment:
This should also do some audit logging on the fulfilling cluster since the auth fails, the only difference here is that the query cluster ignores the problem.